### PR TITLE
Lowess and PolyFitWithCI updates

### PR DESCRIPTION
**Pull request by Ofosu-Osei**
This is an initial push of the current update for your review. I'll deploy to pypi if no changes are needed for now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,38 +79,21 @@ def test_line_label(sample_data, cleanup_files):
 ```python
 import seaborn_objets_recipes as sor
```
Comment (nickeubank) [inline README.md:80]: typo there

File: README.md [hunk 2]
```diff
@@ -37,7 +36,7 @@ Please refer to the [testing file](https://github.com/Ofosu-Osei/seaborn_objects
...
```
Comment (nickeubank) [inline README.md:37]: typo

File: README.md [hunk 3]
```diff
@@ -125,27 +106,23 @@ def test_lowess():
...
```
Comment (nickeubank) [inline README.md:107]: typo

File: README.md [hunk 4]
```diff
@@ -55,15 +54,14 @@ def test_line_label(sample_data, cleanup_files):
...
-        # Will be able to do so.Line() + LineLabel().
+        
         .add(
             sor.LineLabel(offset=5),
             so.Agg(),
-            sor.Rolling(window_type="gaussian", window_kwargs={"std": 2}),
+            rolling,
             legend=False,
         )
         .add(so.Band(), so.Est(errorbar="se"), legend=False)
```
Comment (nickeubank) [inline README.md:67]: I don't see a band in the figure below — is there supposed to be one, and if not, what's this line doing?

**Comment by nickeubank** [comment]
I'd suggest taking all the code snippets in the README and make sure they run — I think there are a few issues there?

File: README.md (hunk 1 above)
Comment (Ofosu-Osei) [reply README.md:80]: I see, thanks!

**Comment by nickeubank** [comment]
Other than that I think we're set! I'll make an issue with next steps

File: README.md (hunk 4 above)
Comment (Ofosu-Osei) [reply README.md:67]: I agree, I think I took it out in one of my actual testing but forgot it here. I'll take it out.

**Comment by Ofosu-Osei** [comment]
> I'd suggest taking all the code snippets in the README and make sure they run — I think there are a few issues there?

So there is a reference to a complete testing processes at the beginning of the usage section. However, I can  replicate the complete code here if necessary.

**Comment by nickeubank** [comment]
Users -- some of whom won't have a good sense of testing -- will want to see examples so I think what's there is good -- you just wanna make sure it works

**Pull request merged**